awscli/customizations/s3/filegenerator.py

```diff
@@ -324,4 +326,4 @@ def _list_single_object(self, s3_path):
         file_size = int(response['ContentLength'])
         last_update = parse(response['LastModified'])
         last_update = last_update.astimezone(tzlocal())
-        return s3_path, file_size, last_update
```

Comment:
I kept it like this to keep parity with the output for local file listings. There is no response associated with a local listing. I will probably break up the logic such that the listing of files and listing of s3 objects do not have the same interface.